onnxruntime/test/onnx/main.cc

```diff
@@ -1154,6 +1252,43 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
     broken_tests.insert({"sce_sum_expanded", "result differs"});
     broken_tests.insert({"sce_sum_log_prob", "result differs"});
     broken_tests.insert({"sce_sum_log_prob_expanded", "result differs"});
+    // float 8, not implemented yet
+    broken_tests.insert({"cast_FLOAT8E5M2FNUZ_to_FLOAT", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_FLOAT8E5M2FNUZ_to_FLOAT16", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_FLOAT8E5M2_to_FLOAT", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_FLOAT8E5M2_to_FLOAT16", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_FLOAT_to_FLOAT8E4M3FN", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_FLOAT_to_FLOAT8E4M3FNUZ", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_FLOAT_to_FLOAT8E5M2", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_FLOAT_to_FLOAT8E5M2FNUZ", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_no_saturate_FLOAT16_to_FLOAT8E4M3FN", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_no_saturate_FLOAT16_to_FLOAT8E4M3FNUZ", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_no_saturate_FLOAT16_to_FLOAT8E5M2", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_no_saturate_FLOAT16_to_FLOAT8E5M2FNUZ", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_no_saturate_FLOAT_to_FLOAT8E4M3FN", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_no_saturate_FLOAT_to_FLOAT8E4M3FNUZ", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_no_saturate_FLOAT_to_FLOAT8E5M2", "float 8 not implemented yet"});
+    broken_tests.insert({"cast_no_saturate_FLOAT_to_FLOAT8E5M2FNUZ", "float 8 not implemented yet"});
+    broken_tests.insert({"castlike_FLOAT8E4M3FNUZ_to_FLOAT", "float 8 not implemented yet"});
```

Comment:
I think we really only need to add these float8 test cases in one place to skip them. though not quite sure when one should indicate a test is disabled vs. broken.

Reply:
If I were to chose, I'll chose disable. I'll do that.

Reply:
It should be fixed. Waiting for the CI to complete.

Reply:
done